stablehlo/reference/tests/add.mlir

```diff
@@ -0,0 +1,270 @@
+// RUN: stablehlo-interpreter-runner -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: add_op_test_f16
+// CHECK-NEXT:  tensor<2x5xf16>
+// CHECK-NEXT:   0.000000e+00 : f16
+// CHECK-NEXT:   -0.000000e+00 : f16
+// CHECK-NEXT:   8.000000e+00 : f16
+// CHECK-NEXT:   8.750000e-01 : f16
+// CHECK-NEXT:   3.999020e-01 : f16
+// CHECK-NEXT:   6.281250e+00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+// CHECK-NEXT:   0xFC00 : f16
+// CHECK-NEXT:   0x7C00 : f16
+
+func.func @add_op_test_f16() -> tensor<2x5xf16> {
+  %0 = stablehlo.constant dense<[[0.0, -0.0, 1.0, 0.125, 0.1], [3.141, 0x7C00, 0x7C00, 0xFC00, 0x7C00]]> : tensor<2x5xf16>
```

Comment:
How about turning these constants into 1D tensors since this is an elementwise op anyway? This will simplify reading and maintaining these tests.